### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by fraliv13**

### CODE REVIEW

File: packages/pure-rules-react/README.md [hunk 1]
```diff
@@ -15,7 +15,23 @@ The library provides three hooks:
 
 
 ## useRulesEngine hook
+React hook that applies the buisiness rules and keeps track of user modified values (dirty field info).
```
Comment (oncicaradupopovici) [inline packages/pure-rules-react/README.md]: typo business

File: packages/pure-rules-react/README.md [hunk 2]
```diff
@@ -15,7 +15,23 @@ The library provides three hooks:
...
+* Return values:
+  1. A stateful rule application result (changed model) 
+  2. A dirty info object to track the model changes
+  3. A function that sets a value for the given property path 
+  4. A function that resets the rule engine state.
+
+Usage example:
+
 ```jsx
+import { useRulesEngine } from "@totalsoft/pure-rules-react";
```
Comment (oncicaradupopovici) [inline packages/pure-rules-react/README.md]: useRulesEngine or usePureRules? I don't know....

File: packages/pure-rules-react/README.md [hunk 3]
```diff
@@ -46,7 +62,20 @@ const SomeComponent = props => {
 ```
 
 ## useRulesEngineProfunctor hook
+React hook that applies the buisiness rules and keeps track of user modified values (dirty field info).
```
Comment (oncicaradupopovici) [inline packages/pure-rules-react/README.md]: typo business

File: packages/pure-rules-react/README.md (hunk 2 above)
Comment (fraliv13) [reply packages/pure-rules-react/README.md]: renamed to useRules / useRulesProfunctor

**Pull request merged**